docs/architecture/grpc-for-wcf-developers/kubernetes.md

```diff
@@ -61,31 +61,31 @@ Kubernetes clusters are designed to scale to hundreds or thousands of nodes and
 
 If you're running Docker Desktop for Windows or Docker Desktop for Mac, Kubernetes is already available. Just enable it in the **Kubernetes** section of the **Settings** window:
 
-![Enable Kubernetes in Docker Desktop](media/kubernetes/enable-kubernetes-docker-desktop.png)
+![Enable Kubernetes in Docker Desktop](media/kubernetes/enable-kubernetes-docker-desktop-v2.png)
```

Comment:
Is *media/kubernetes/enable-kubernetes-docker-desktop.png* needed anymore, if not was it deleted?

Reply:
`enable-kubernetes-docker-desktop.png` - is no more needed. So I have deleted it.